assets/app/view/game/entities.rb

```diff
@@ -17,13 +17,13 @@ def render
             gap: '3rem 1.2rem',
           },
         }
-
         players = @game.players
         if (i = players.map(&:name).rindex(@user&.dig(:name)))
           players = players.rotate(i)
         end
 
-        player_owned, bank_owned = (@game.corporations + @game.minors).sort_by(&:name).partition(&:owner)
+        entity_candidates = (@game.corporations + @game.minors).reject(&:closed?)
+        player_owned, bank_owned = entity_candidates.sort_by(&:name).partition(&:owner)
```

Comment:
you can just do this all in one line

x, y = (...)
.rject(&:closed?)
.sort_by
.partition

Reply:
Fixed